tensorflow_lite_support/ios/task/vision/sources/TFLImageClassifier.h

```diff
@@ -50,7 +50,7 @@ NS_ASSUME_NONNULL_BEGIN
  * @return An instance of TFLImageClassifierOptions set to the specified
  * modelPath.
  */
-- (nullable instancetype)initWithModelPath:(NSString *)modelPath;
+- (instancetype)initWithModelPath:(NSString *)modelPath;
```

Comment:
@lu-wang-g @mbrenon After implementing the Python API, I realized that this default initializer should be implemented in the ImageClassifer rather than the ImageClassifierOptions. So we can do a one-liner:

```objc
TFLImageClassifier *classifier = [TFLImageClassifier imageClassifierWithModelPath:@"model.tflite" error:nil];
```

rather than 2 liner:
```objc
TFLImageClassifierOptions options = [[TFLImageClassifierOptions alloc] initWithModelPath:@"model.tflite"];
TFLImageClassifier *classifier = [TFLImageClassifier imageClassifierWithOptions:@"model.tflite" error:nil];
```

This will align better with the Python and Java API. WDYT?

Reply:
Right now TFLImageClassifierOptions is not a property or instance variable of TFLImageClassifier. So with the suggested method signature we will have to add the image classifier options as a readonly property, since our C Image Classifier initialisation will be completed once the ObjC Image Classifier init completes and we can't allow changes after that. 
So if this signature is to be used we will need 2 init methods:
1. TFLImageClassifier *classifier = [TFLImageClassifier imageClassifierWithModelPath:@"model.tflite" error:nil];
If you use this initialiser you won't be able to modify any other options. It will be like using default options with a modelPath. 
2.  TFLImageClassifier *classifier = [TFLImageClassifier imageClassifierWithOptions:options error:nil];
This will let us configure options acc convenience and init. From thereon, options will be only readable.

Reply:
@priankakariatyml Yes that's what I expect it to be. Let's see what Lu and Maxime think about this new API. If they agrees, let's migrate the API so that we'll reduce the amount of code to write for users, and also have consistency between languages.